### ISSUE TITLE
Failed to start application with quarkus-cache:1.11.0.Final

### ISSUE THREAD

**Issue by SuperPDX** [quarkusio/quarkus#14436]
ERROR [io.qua.run.Application] (Quarkus Main Thread) Failed to start application (with profile dev): 

java.lang.NullPointerException
	at io.quarkus.cache.runtime.CaffeineCacheSupplier.get(CaffeineCacheSupplier.java:21)
	at io.quarkus.cache.runtime.CaffeineCacheSupplier.get(CaffeineCacheSupplier.java:15)
	at io.quarkus.vertx.http.runtime.devmode.DevConsoleRecorder.addInfo(DevConsoleRecorder.java:37)
	at io.quarkus.deployment.steps.DevConsoleProcessor$runtimeTemplates-308161071.deploy_0(DevConsoleProcessor$runtimeTemplates-308161071.zig:95)
	at io.quarkus.deployment.steps.DevConsoleProcessor$runtimeTemplates-308161071.deploy(DevConsoleProcessor$runtimeTemplates-308161071.zig:40)
	at io.quarkus.runner.ApplicationImpl.doStart(ApplicationImpl.zig:818)
	at io.quarkus.runtime.Application.start(Application.java:90)
	at io.quarkus.runtime.ApplicationLifecycleManager.run(ApplicationLifecycleManager.java:97)
	at io.quarkus.runtime.Quarkus.run(Quarkus.java:66)
	at io.quarkus.runtime.Quarkus.run(Quarkus.java:42)
	at io.quarkus.runtime.Quarkus.run(Quarkus.java:119)

**Comment by gwenneg** [comment]
Thanks for reporting this @SuperPDX!

We already fixed that bug but unfortunately the commit could not be included into `1.11.0.Final`. The next release will certainly include the fix though. In the meantime, if you add a single cache annotation in your application, the NPE should no longer be thrown:

```
import javax.enterprise.context.ApplicationScoped;
import io.quarkus.cache.CacheInvalidateAll;

/*
 * This is a workaround for a NPE in CaffeineCacheSupplier with Quarkus 1.11.0.Final.
 * Delete this class when Quarkus is upgraded to 1.11.1.Final or higher.
 */
@ApplicationScoped
public class CacheNpeWorkaround {
    @CacheInvalidateAll(cacheName = "npe-workaround")
    public void invalidateAll() {
    }
}
```

**Comment by gsmet** [comment]
This will be fixed in the upcoming 1.11.1.Final.
